widget/scroller.go

```diff
@@ -371,10 +372,20 @@ func (s *ScrollContainer) Dragged(e *fyne.DragEvent) {
 
 // MinSize returns the smallest size this widget can shrink to
 func (s *ScrollContainer) MinSize() fyne.Size {
+	if s.minSize.Width > 0 && s.minSize.Height > 0 {
```

Comment:
```suggestion
	if s.minSize != fyne.Size{} {
```

Reply:
P.S.: Yes, this is different to prior test but I think it meets expectations better.

Reply:
I had to revert this, it doesn't compile